api/src/test/java/com/epam/pipeline/test/web/AbstractControllerTest.java

```diff
@@ -197,7 +197,8 @@ public void performRequestWithoutResponse(final MockHttpServletRequestBuilder re
     public MultiValueMap<String, String> multiValueMapOf(Object... objects) {
         final MultiValueMap<String, String> map = new LinkedMultiValueMap<>();
         for (int i = 0; i < objects.length; i += 2) {
-            map.add(String.valueOf(objects[i]), String.valueOf(objects[i + 1]));
+            map.add(String.valueOf(objects[i]).replaceAll("[\\[\\]]", ""),
+                    String.valueOf(objects[i + 1]).replaceAll("[\\[\\]]", ""));
```

Comment:
The current approach is error-prone. Imagine if some controller method accepts just a string and not a list. If we try to pass f.e. the following string `Array[Sample]` to the method using `multiValueMapOf` it will be cut to `ArraySample`. Imagine debugging something like this :)

As long as this special processing is required for only `List` type then we can handle it explicitly using `instanceof`. Additionally I assume there is no need to process the map keys. Probably we can even check that the keys are of `String` type and fail otherwise.